js/base.js

```diff
@@ -89,6 +89,18 @@ export class API {
         await this.consumer.consume(topic, { callback: callback, ...options });
     }
 
+    /**
+     * Lists the available topics.
+     *
+     * @param {Function|Object} options Object that includes configuration variables.
+     * @returns {String[]} The topic names.

```

Comment:
In vanilla javascript we don't go so far with the typing
```suggestion
     * @returns {Array} The topic names.
```